gui/src/app/state/settings/mod.rs

```diff
@@ -78,7 +78,12 @@ impl State for SettingsState {
             }
             Message::View(view::Message::Settings(view::SettingsMessage::EditWalletSettings)) => {
                 self.setting = Some(
-                    WalletSettingsState::new(self.data_dir.clone(), self.wallet.clone()).into(),
+                    WalletSettingsState::new(
+                        self.data_dir.clone(),
+                        self.wallet.clone(),
+                        daemon.get_info().unwrap().timestamp,
```

Comment:
not sure about unwrap here?
buble up error?
change creation_date type as Option<u32>?

Reply:
Daemon request should be in an asynchronous iced command. But I think for this time it is better to pass the wallet timestamp to the Wallet struct in src/app/wallet.rs
```
impl Wallet {
    pub fn new(main_descriptor: LianaDescriptor, timestamp: u32) -> Self {

```